[2022-02-09 09:01:17,626] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-10-01T23:00:00+00:00 [queued]>
[2022-02-09 09:01:17,639] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-10-01T23:00:00+00:00 [queued]>
[2022-02-09 09:01:17,639] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:01:17,639] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:01:17,640] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:01:17,657] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2021-10-01 23:00:00+00:00
[2022-02-09 09:01:17,663] {standard_task_runner.py:52} INFO - Started process 534 to run task
[2022-02-09 09:01:17,670] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2021-10-01T23:00:00+00:00', '--job-id', '513', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingestee_gcs_dag.py', '--cfg-path', '/tmp/tmp_60olt2q', '--error-file', '/tmp/tmpzzusvzoi']
[2022-02-09 09:01:17,672] {standard_task_runner.py:77} INFO - Job 513: Subtask local_to_gcs_task
[2022-02-09 09:01:17,761] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-10-01T23:00:00+00:00 [running]> on host cc021fb126ed
[2022-02-09 09:01:17,828] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:01:17,866] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-01T23:00:00+00:00
[2022-02-09 09:01:18,409] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:01:18,423] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20211001T230000, start_date=20220209T090117, end_date=20220209T090118
[2022-02-09 09:01:18,447] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:01:18,490] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-02-09 09:17:05,617] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-10-01T23:00:00+00:00 [queued]>
[2022-02-09 09:17:05,649] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-10-01T23:00:00+00:00 [queued]>
[2022-02-09 09:17:05,650] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:17:05,650] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-02-09 09:17:05,651] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-09 09:17:05,674] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2021-10-01 23:00:00+00:00
[2022-02-09 09:17:05,693] {standard_task_runner.py:52} INFO - Started process 703 to run task
[2022-02-09 09:17:05,703] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'yelloTaXi_HW_data_ingestee_gcs_dag', 'local_to_gcs_task', 'scheduled__2021-10-01T23:00:00+00:00', '--job-id', '617', '--raw', '--subdir', 'DAGS_FOLDER/yelloTaXi_data_ingesteee_gcs_dag.py', '--cfg-path', '/tmp/tmp49ptm2nx', '--error-file', '/tmp/tmpgxe_2jla']
[2022-02-09 09:17:05,705] {standard_task_runner.py:77} INFO - Job 617: Subtask local_to_gcs_task
[2022-02-09 09:17:05,854] {logging_mixin.py:109} INFO - Running <TaskInstance: yelloTaXi_HW_data_ingestee_gcs_dag.local_to_gcs_task scheduled__2021-10-01T23:00:00+00:00 [running]> on host 75953ff4f3d5
[2022-02-09 09:17:05,918] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-02-09 09:17:05,946] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yelloTaXi_HW_data_ingestee_gcs_dag
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2021-10-01T23:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-01T23:00:00+00:00
[2022-02-09 09:17:06,477] {python.py:175} INFO - Done. Returned value was: None
[2022-02-09 09:17:06,492] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=yelloTaXi_HW_data_ingestee_gcs_dag, task_id=local_to_gcs_task, execution_date=20211001T230000, start_date=20220209T091705, end_date=20220209T091706
[2022-02-09 09:17:06,532] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-09 09:17:06,570] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
